utils/wasm-instrument/src/syscalls.rs

```diff
@@ -285,6 +287,12 @@ impl SysCallName {
                 SysCallSignature::gr([Ptr(PtrInfo::new_mutable(PtrType::ErrorWithSignalCode))])
             }
             Self::MessageId => SysCallSignature::gr([Ptr(PtrInfo::new_mutable(PtrType::Hash))]),
+            Self::ExecSettings => SysCallSignature::gr([
+                // The PtrType::BufferStart doesn't work here as it requires length_param_idx
+                // Should we introduce something like PtrType::Buffer
+                Ptr(PtrInfo::new_mutable(PtrType::BlockNumber)),
+                Size,
```

Comment:
```suggestion
                I32,
```

Reply:
There is no such variant in the `ParamType` enum

Reply:
@SabaunT, I guess we need to introduce new parameter type?

Reply:
> @SabaunT, I guess we need to introduce new parameter type?

I'd add something like `Version`

Reply:
It's never a problem to code something new xd